src/Fixer/Phpdoc/PhpdocSeparationFixer.php

```diff
@@ -26,35 +31,59 @@
 
 /**
  * @author Graham Campbell <hello@gjcampbell.co.uk>
+ * @author Jakub Kwaśniewski <jakub@zero-85.pl>
  */
-final class PhpdocSeparationFixer extends AbstractFixer
+final class PhpdocSeparationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
+    private TagComparator $tagComparator;
+
+    private bool $standardTagsOnly = true;
+
     /**
      * {@inheritdoc}
      */
     public function getDefinition(): FixerDefinitionInterface
     {
-        return new FixerDefinition(
-            'Annotations in PHPDoc should be grouped together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line.',
-            [
-                new CodeSample(
-                    '<?php
+        $code = <<<'EOF'
+<?php
 /**
- * Description.
- * @param string $foo
+ * Hello there!
  *
+ * @author John Doe
+ * @custom Test!
+ *
+ * @throws Exception|RuntimeException foo
+ * @param string $foo
  *
  * @param bool   $bar Bar
- * @throws Exception|RuntimeException
- * @return bool
+ * @return int  Return the number of changes.
  */
-function fnc($foo, $bar) {}
-'
-                ),
-            ]
+
+EOF;
+
+        return new FixerDefinition(
+            'Annotations in PHPDoc should be grouped together so that annotations of the same type immediately follow each other. Annotations of a different type are separated by a single blank line, except those specified in `additional_groups` option.',
```

Comment:
This needs to be updated after the `additional_groups` option has been removed.

Reply:
Done!